__tests__/parsers.test.ts

```diff
@@ -100,6 +104,60 @@ describe("Parse output", () => {
   });
 });
 
+describe("parseFunctionCall", () => {
+  it("hyphenated argument name", () => {
+    const str = `get_account(gtmhub-accountId="64b17ac6548041a751aaf2f6", id_team="64b17ac6548041a751aaf2f7")`;
+    const output = parseFunctionCall(str);
+    const expectedOutput = {
+      name: "get_account",
+      args: {
+        "gtmhub-accountId": "64b17ac6548041a751aaf2f6",
+        id_team: "64b17ac6548041a751aaf2f7",
+      },
+    };
+    expect(output).toStrictEqual(expectedOutput);
+  });
+  it("correctly parses function with floating point argument", () => {
+    const str = `set_coordinates(x=3.14, y=0.98)`;
+    const output = parseFunctionCall(str);
+    const expectedOutput = {
+      name: "set_coordinates",
+      args: { x: 3.14, y: 0.98 },
+    };
+    expect(output).toEqual(expectedOutput);
+  });
+  it("correctly parses function mixed argument types", () => {
+    const str = `set_coordinates(x=3.14, placeName="The Moon", y=0.98)`;
+    const output = parseFunctionCall(str);
+    const expectedOutput = {
+      name: "set_coordinates",
+      args: { x: 3.14, y: 0.98, placeName: "The Moon" },
+    };
+    expect(output).toEqual(expectedOutput);
+  });
+  it("hyphenated function name", () => {
+    const str = `set-coordinates(x=3.14, placeName="The Moon", y=0.98)`;
+    const output = parseFunctionCall(str);
+    const expectedOutput = {
+      name: "set-coordinates",
+      args: { x: 3.14, y: 0.98, placeName: "The Moon" },
+    };
+    expect(output).toEqual(expectedOutput);
+  });
```

Comment:
Think this isn't allowed by the `/actions` page - if so it's not worth testing :smile: